### ISSUE TITLE
Gitea Issue-Integration

### ISSUE THREAD

**Issue by jonaheinrichs** [johannesjo/super-productivity#533]
Hey there!
At first I would like to say that SuperProductivity is really awesome and the best productivity app I have tested so far.
My Git-Repositories I host via [Gitea](https://github.com/go-gitea/gitea) on my own or company server.
And I would like to use the Issue Integration with Gitea.
Gitea also gives an API to access Issues and so on, like Gitlab & Github.
Would it be much effort to implement it?
Thanks in advance!

**Comment by johannesjo** [comment]
Hey there!

Currently I am not planning to work on implementing Gitea support myself, but I gladly accept any Pull Requests. Depending on how different the Gitea operates compared to GitHub or Gitlab API this might take an experienced developer everything from a couple of hours to a couple of days.

**Comment by alexanderadam** [comment]
Duplicate of #337

**Comment by alexanderadam** [comment]
In case you want to make sure that integration works, you can simply test the API integration on [the _official test instance_ at `https://try.gitea.io/`](https://try.gitea.io/). There you can also find [the _Swagger API_ definition](https://try.gitea.io/api/swagger).
If you rather want to use a _local_ Gitea instance, you might want to use [the official _Docker image_](https://hub.docker.com/r/gitea/gitea/) (there's also an example [`docker-compose.yml`](https://hub.docker.com/r/gitea/gitea/#quick-set-up)).

**Comment by r0bbie** [comment]
I was about to open a feature request for this and spotted there was one already! Commenting to note this would be a great feature to have (I use Gitea for my personal project hosting, and with their recent announcement for federation support going to be interesting to see where the project goes!)

Hope commenting on this closed task is the right thing to do, github-actions comments above seem to suggest it is?

**Comment by hugaleno** [comment]
I'm working on openproject integration now but I will take a look on gitea api. Maybe I can start working on this after.

**Comment by hugaleno** [comment]
You can assign this one to me @johannesjo . I will start working on it after #2005 review

**Comment by hugaleno** [comment]
## Action plan

Basic Features based on existing integrations.
- [ ]  Authentication via api access token 

- [ ] Allow for import of work packages through the add task bar

- [ ]  Add scope  to filter issues 

- [ ] Display basic issue data (local properties) in side panel

## Later on

- [ ] Check imported work packages for updates for active list

- [ ] Transition handling

- [ ] Add worklog submit via Dialog when a task is marked as done

**Comment by johannesjo** [comment]
Please let me know if you need anything @hugaleno ! This is great!

**Comment by johannesjo** [comment]
Thanks to @hugaleno this is now available!
